{{ cookiecutter.project_slug }}/.gitignore

```diff
@@ -0,0 +1,188 @@
+######## Below is auto generated .gitignore file for Python projects. #########
```

Comment:
Replace with:

> This section contains auto-generated .gitignore patterns for Python projects.
> Add project-specific patterns below this section.

Also applies to the `.gitignore` for the Cookiecutter project.